### PR TITLE
feat(): Add a Gitter chat badge to README.md

### PR DESCRIPTION
**Pull request by Lordnoname**
Add a Gitter chat badge to README.md

### CODE REVIEW

**Comment by Lordnoname** [comment]
@DevVersion please review

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-### Werdenfels-Gymnasium Hilfe [![BuildStatus](https://travis-ci.org/Werdenfels-Gymnasium/WG-Hilfe.svg?branch=master)](https://travis-ci.org/Werdenfels-Gymnasium/WG-Hilfe)
+### Werdenfels-Gymnasium Hilfe [![BuildStatus](https://travis-ci.org/Werdenfels-Gymnasium/WG-Hilfe.svg?branch=master)](https://travis-ci.org/Werdenfels-Gymnasium/WG-Hilfe)[![Join the chat at https://gitter.im/Werdenfels-Gymnasium/WG-Hilfe](https://badges.gitter.im/Werdenfels-Gymnasium/WG-Hilfe.svg)](https://gitter.im/Werdenfels-Gymnasium/WG-Hilfe?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```
Comment (DevVersion) [inline README.md]: Can you add a space between that two badges?

**Comment by Lordnoname** [comment]
@DevVersion Can I merge now or do you want to merge?

**Comment by DevVersion** [comment]
LGTM. Yes you can merge but never **FORCE** Push

**Comment by Lordnoname** [comment]
Okay :stuck_out_tongue:

**Pull request closed without merging**